_data/whats-new.yml

```diff
@@ -4,8 +4,52 @@ description: |
   We exclude from this list proofreading, spelling checks, and all minor updates.
 link: /whats-new.html
 thread: /whatsnew-feed.xml
-updated: Wed Mar 27 18:03:03 2019
+updated: Tue Apr  2 11:41:12 2019
 entries:
+- description: Added a topic about [how to retrieve](https://devdocs.magento.com/guides/v2.3/graphql/get-customer-authorization-token.html)
+    the customer's authorization token for GraphQL queries and mutations.
+  versions: 2.3.1
+  type: New topic
+  date: April  1, 2019
+  link: https://github.com/magento/devdocs/pull/4052
+- description: Added section about mutations in [Resolvers](https://devdocs.magento.com/guides/v2.3/graphql/develop/resolvers.html)
```

Comment:
The previous item's sentence has ending punctuation but this one does not. I advise reconciling punctuation.

Reply:
@erikmarr please consider the comment above for #4052

Reply:
@keharper please consider to apply the above comment for #4096

Reply:
@dshevtsov PR 4052 corresponds to the description in line 9 which contains an ending period. It looks like we're fairly consistent with periods for what's new entries:
https://devdocs.magento.com/whats-new.html
My vote for reconciliation is to use periods. :-)

Reply:
@eriksmarr, yes I removed the comment that mentioned your PR.
Sorry for confusion.

Reply:
@shrielenee reconciled